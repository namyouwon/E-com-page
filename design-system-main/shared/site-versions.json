{
  "2.16": {
    "site_version": "v1.2.31",
    "nvm_version": 14
  },
  "2.15": {
    "site_version": "v1.2.30",
    "nvm_version": 14
  },
  "2.14": {
    "site_version": "v1.2.28",
    "nvm_version": 10
  },
  "2.13": {
    "site_version": "v1.2.24",
    "nvm_version": 10
  },
  "2.12": {
    "site_version": "v1.2.21",
    "nvm_version": 10
  },
  "2.11": {
    "site_version": "v1.2.21",
    "nvm_version": 10
  },
  "2.10": {
    "site_version": "v1.2.11",
    "nvm_version": 10
  },
  "2.9": {
    "site_version": "v1.2.6",
    "nvm_version": 8
  },
  "2.8": {
    "site_version": "v1.2.3",
    "nvm_version": 8
  },
  "2.7": {
    "site_version": "v1.0.0",
    "nvm_version": 8
  }
}
