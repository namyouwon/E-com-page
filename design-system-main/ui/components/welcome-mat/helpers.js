// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React, { Fragment } from 'react';
import PropTypes from 'prop-types';

// local components
import { Button } from '../buttons/base/example';
import { Checkbox } from '../checkbox/base/example';
import { FormElement } from '../form-element/index';

export const exampleDemoStyles = 'height: 800px;';

export const WelcomeMatContext = props => (
  <div className="demo-only" style={{ height: '800px' }}>
    {props.children}
  </div>
);

WelcomeMatContext.propTypes = {
  children: PropTypes.node.isRequired
};

/**
 * Sample Tile Data
 */
export const sampleTiles = () => [
  {
    symbol: 'animal_and_nature',
    title: 'Welcome to Salesforce!',
    description: 'Lorem ipsum dolor sit amet, lorem ipsum dolor sit amet.'
  },
  {
    symbol: 'call',
    title: 'Learn About OpenCTI',
    description: 'Lorem ipsum dolor sit amet, lorem ipsum dolor sit amet.'
  },
  {
    symbol: 'upload',
    title: 'Power Up the Utility Bar',
    description:
      'Tap into case history or share notes with fellow agentsâ€”it all happens on the utility bar.'
  },
  {
    symbol: 'magicwand',
    title: 'Customize your view',
    description:
      "Tailor your cases to your team's workflow with custom list views."
  },
  {
    symbol: 'knowledge_base',
    title: 'Share the Knowledge',
    description:
      "Harness your team's collective know-how with our powerful knowledge base."
  }
];

/**
 * Sample Actions
 */
export const WelcomeMatSampleActions = () => (
  <Fragment>
    <Button isBrand>Learn More</Button>
    <div className="slds-m-top_large">
      <FormElement>
        <Checkbox label="Don't show this again" />
      </FormElement>
    </div>
  </Fragment>
);

/**
 * Welcome Mat Info Title Element
 *
 * @param {*} props
 * @prop {string} labelId - aria label id
 * @prop {object} children - children elements
 */
export const WelcomeMatInfoTitle = props => (
  <h2 className="slds-welcome-mat__info-title" id={props.labelId}>
    {props.children}
  </h2>
);

WelcomeMatInfoTitle.propTypes = {
  labelId: PropTypes.string.isRequired,
  children: PropTypes.string.isRequired
};

/**
 * Welcome Mat Info Description Element
 *
 * @param {*} props
 * @prop {object} children - children elements
 */
export const WelcomeMatInfoDesc = props => (
  <div className="slds-welcome-mat__info-description slds-text-longform">
    {props.children}
  </div>
);

WelcomeMatInfoDesc.propTypes = {
  children: PropTypes.node.isRequired
};

/**
 * Welcome Mat Info Actions Container
 *
 * @param {*} props
 * @prop {object} children - children elements
 */
export const WelcomeMatInfoActions = props => (
  <div className="slds-welcome-mat__info-actions">
    <WelcomeMatSampleActions />
  </div>
);

WelcomeMatInfoActions.propTypes = {
  children: PropTypes.node
};

/**
 * Welcome Mat Marketing Content Example
 *
 * @param {*} props
 * @prop {string} labelId
 * @prop {string} headline
 * @prop {object} description
 * @prop {object} actions
 */
export const WelcomeMatMarketingContent = props => {
  const { labelId, headline, description, actions } = props;

  return (
    <Fragment>
      <WelcomeMatInfoTitle labelId={labelId}>{headline}</WelcomeMatInfoTitle>
      <WelcomeMatInfoDesc>{description}</WelcomeMatInfoDesc>
      <WelcomeMatInfoActions>{actions}</WelcomeMatInfoActions>
    </Fragment>
  );
};

WelcomeMatMarketingContent.propTypes = {
  labelId: PropTypes.string.isRequired,
  headline: PropTypes.string.isRequired,
  description: PropTypes.object,
  actions: PropTypes.object
};

WelcomeMatMarketingContent.defaultProps = {
  headline: 'The Lightning Experience is here!',
  description: (
    <p>
      Welcome to Lightning Experience, the modern, beautiful user experience
      from Salesforce. With a sales-and service-centric mindset, we focused on
      reinventing the desktop environment to better support your business
      processes.
    </p>
  )
};
