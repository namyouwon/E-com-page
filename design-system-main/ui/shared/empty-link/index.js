import React from 'react';
import PropTypes from 'prop-types';

// Prevents React warning about javascript:void(0) usage
// https://github.com/facebook/react/pull/15047
const EmptyLink = ({ children, title }) => (
  <a href="#" onClick={e => e.preventDefault()} title={title}>
    {children}
  </a>
);

EmptyLink.propTypes = {
  children: PropTypes.string,
  title: PropTypes.string
};

export default EmptyLink;
