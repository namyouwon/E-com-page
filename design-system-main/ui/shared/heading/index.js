import React from 'react';
import PropTypes from 'prop-types';

const elements = {
  1: 'h1',
  2: 'h2',
  3: 'h3',
  4: 'h4',
  5: 'h5',
  6: 'h6'
};

function Heading({ level, children, ...props }) {
  return React.createElement(elements[level] || elements[2], props, children);
}

Heading.propTypes = {
  level: PropTypes.oneOf(Object.keys(elements)),
  children: PropTypes.node
};

export default Heading;
