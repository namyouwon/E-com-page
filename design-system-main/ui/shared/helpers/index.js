// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import omit from 'lodash.omit';
import range from 'lodash.range';
import reverse from 'lodash.reverse';
import startCase from 'lodash.startcase';
import times from 'lodash.times';
import truncate from 'lodash.truncate';
import upperFirst from 'lodash.upperfirst';
import IsDependentOn from './prop-types/is-dependent-on';
import CannotBeSetWith from './prop-types/cannot-be-set-with';
import StoryWrapper from '../../../shared/components/StoryWrapper';

import { storiesOf } from '@storybook/react';
import StoryFrame from '../../../shared/components/StoryFrame';
import DocsPage from '../../../.storybook/components/DocsPage';

// SHIM Lodash because it caches in node_modules and generates id's that are always incrementing
const uniqueId = (() => {
  const PREFIXES = {};
  let idCounter = 0;
  const newCounter = prefix => {
    PREFIXES[prefix] = 0;
  };
  const incCounter = prefix => {
    PREFIXES[prefix] = PREFIXES[prefix] + 1;
    return PREFIXES[prefix];
  };
  const initCounterForPrefix = prefix =>
    PREFIXES[prefix] != null ? PREFIXES[prefix] : newCounter(prefix);
  const addToPrefix = prefix => {
    initCounterForPrefix(prefix);
    return prefix + incCounter(prefix);
  };
  return prefix => (prefix ? addToPrefix(prefix) : idCounter++);
})();

// Get a component example object from the exported states and examples objects
// To retrieve a specific property value from the object, pass in its key as an argument
export const getDisplayExampleById = (collection, id, key) => {
  if (
    !(
      Array.isArray(collection) &&
      collection.every(
        o => o.hasOwnProperty('id') && React.isValidElement(o.element)
      )
    )
  )
    throw new Error(
      `${JSON.stringify(
        collection,
        null,
        2
      )} has broken schema that requires an array of example/state objects with 'id' and 'element' properties`
    );

  // if an ID was provided then search for it in the collection
  if (id !== undefined) {
    const elementObj = collection.filter(example => example.id === id);

    if (elementObj && elementObj[0]) {
      return elementObj[0][key] || elementObj[0];
    } else {
      throw new Error(`No display element with id "${id}" found`);
    }
  } else {
    // if no ID was provided then we simply return the first item of the array
    return collection[0][key];
  }
};

// Get a component example in doc block from the exported states and examples objects
export const getDisplayElementById = (collection, id) => {
  // if collection is an array, continue as expected
  if (Array.isArray(collection)) {
    return getDisplayExampleById(collection, id, 'element');
  }

  // if collection is not an array simply return it (probably a React element)
  return collection;
};

// Get a component example's styles in doc block from the exported states and examples objects
export const getDemoStylesById = (collection, id) =>
  getDisplayExampleById(collection, id, 'demoStyles');

export const capitalize = str => {
  if (typeof str === 'string') {
    return str.replace(/^\w/, c => c.toUpperCase());
  } else {
    return '';
  }
};

function union(setA, setB) {
  let _union = new Set(setA);
  for (let elem of setB) {
    _union.add(elem);
  }
  return _union;
}

/**
 * @desc Get all contexts for examples for a single component
 * @param object $object - the object to get contexts from
 * @return array - array of example contexts
 */
export const getExampleContexts = object => {
  let contexts = [];

  object.map(group => {
    const groupSet = new Set(
      group.map(example =>
        example.context === undefined ? 'kitchen' : example.context
      )
    );
    contexts = [...union(groupSet, new Set(contexts))];
  });

  return contexts;
};

/**
 * @desc Get all examples for a single component by type
 * @param object $object - the object to check types against
 * @param array $types - the type of examples you want
 * @return array - array of component examples based parameters
 */
export const getDisplayCollectionsByType = (object, types) => {
  let collection = [];
  if (Array.isArray(types)) {
    types.map(type => {
      if (object.hasOwnProperty(type)) {
        if (Array.isArray(object[type])) {
          object[type].map(element => {
            const {
              id,
              demoStyles,
              storybookStyles,
              label,
              demoProps,
              deprecated,
              context
            } = element;
            return collection.push({
              id: id,
              component: getDisplayElementById(object[type], id),
              label,
              demoStyles,
              storybookStyles,
              demoProps,
              deprecated,
              context
            });
          });
        } else {
          collection.push({
            label: 'Default',
            component: object[type]
          });
        }
      }
    });
  } else {
    throw new Error(`Expected "${types}" to be an array`);
  }
  return collection;
};

/**
 * @desc Get all examples for multiple components by type
 * @param array $array - the components to check types against
 * @param array $types - the type of examples you want
 * @return array - array of component examples based on parameters
 */
export const getAllDisplayCollectionsByType = (array, types) => {
  let collection = [];
  if (Array.isArray(array)) {
    array.map(element =>
      collection.push(getDisplayCollectionsByType(element, types))
    );
  } else {
    throw new Error(`Expected "${array}" to be an array`);
  }
  return collection;
};

/**
 * @desc Get the StoryWrapper for a demo-styled example as a Story decorator
 * @param object $example - the example object being Story-ified
 * @return object - decorator object to pass to storiesOf.add
 */
export const getStoryWrapperDecorator = example => {
  const { demoStyles, demoProps } = example;
  return demoStyles || demoProps
    ? {
        decorators: [
          storyFn => (
            <StoryWrapper styles={demoStyles} {...demoProps}>
              {storyFn()}
            </StoryWrapper>
          )
        ]
      }
    : null;
};

/**
 * @desc Get the parameters for a story with consideration for example wrapper decorator
 * @param {...Object} $paramObjects - the params to pass to story as Objects
 * @return object - parameter object to pass to storiesOf.add
 */
export const getExampleStoryParams = (...paramObjects) =>
  paramObjects.reduce((prev, next) => {
    if (prev.decorators && next.decorators)
      next.decorators = [...prev.decorators, ...next.decorators];
    return { ...prev, ...next };
  }, {});

/**
 * @desc Return the nubbin position class
 * @param {string} position Accepts declarative location e.g. "top left", 'bottom right', 'top', etc
 * @return Classname as a string
 */
export const getNubbinClass = position => {
  const nubbinPositionArray = position.split(' ');
  const nubbinComputedClass = 'slds-nubbin_' + nubbinPositionArray.join('-');
  return nubbinComputedClass;
};

/**
 * @desc Factory for creating custom React prop-types
 * @param {boolean} isRequired Sets the prop-type to required
 * @param {function} callback Prop-type validation logic
 */
export const createCustomPropType = (isRequired, callback) => {
  // The factory returns a custom prop type
  return function(props, propName, componentName) {
    const prop = props[propName];
    if (prop == null) {
      // Prop is missing
      if (isRequired) {
        // Prop is required but wasn't specified. Throw an error.
        throw new Error();
      }
      // Prop is optional. Do nothing.
    } else {
      callback(props, propName, componentName);
    }
  };
};

/**
 *
 */
export const generateStories = (
  patternName,
  examples,
  collections,
  Docs,
  options = {
    defaultDemoStyles: '',
    isFullBleed: false,
    isViewport: false,
    eyes: undefined
  }
) => {
  // retrieve examples by type
  const kitchenSink = getAllDisplayCollectionsByType(examples, collections);

  // retrieve contexts defined in examples definition files
  const contexts = getExampleContexts(kitchenSink);

  return contexts.map(context => {
    storiesOf(`Components/${patternName}`, module).add(
      `${capitalize(context)} Sink`,
      () =>
        kitchenSink.map((element, idx) => {
          const kitchen = element.filter(el =>
            context === 'kitchen'
              ? el.context === context || el.context === undefined
              : el.context === context
          );

          return kitchen.map(
            ({ label, component, storybookStyles, demoStyles }) => {
              // if storybookStyles is a boolean true then we use the same styles defined in demoStyles
              let storyStyles;
              if (storybookStyles === true) {
                storyStyles = demoStyles || '';
              } else {
                storyStyles = storybookStyles || '';
              }
              storyStyles = options.defaultDemoStyles + storyStyles;

              return (
                <StoryFrame
                  component={component}
                  label={label}
                  key={`${context}-sink-${label}-${idx}`}
                  styles={storyStyles}
                  isFullBleed={options.isFullBleed}
                  isViewport={options.isViewport}
                />
              );
            }
          );
        }),
      {
        docs: {
          page: () => <DocsPage title={patternName} Docs={Docs} />
        },
        eyes: options.eyes
      }
    );
  });
};

export default {
  omit,
  range,
  reverse,
  startCase,
  times,
  truncate,
  uniqueId,
  upperFirst
};

export { IsDependentOn, CannotBeSetWith };
