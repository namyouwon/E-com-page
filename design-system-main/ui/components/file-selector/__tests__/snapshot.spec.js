// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license
/* eslint-env jest */

// import React from 'react';
import createHelpers from '../../../../jest.helpers';
import { getDisplayElementById } from '../../../shared/helpers';
import baseExamples from '../base/example';
import imageExamples from '../image/example';
import integratedExamples from '../integrated/example';

const { matchesMarkup } = createHelpers(__dirname);

describe('File Selector', () => {
  it('renders a default file selector', () =>
    matchesMarkup(getDisplayElementById(baseExamples, 'default'))
  )

  it('renders a file selector with error', () =>
    matchesMarkup(getDisplayElementById(baseExamples, 'file-selector-files-error'))
  )

  it('renders a file selector dragover', () =>
    matchesMarkup(getDisplayElementById(baseExamples, 'file-selector-files-dragover'))
  )

  it('renders a file selector dragover with error', () =>
    matchesMarkup(getDisplayElementById(baseExamples, 'file-selector-files-dragover-error'))
  )
})

describe('File Selector - Images', () => {
  it('renders a default file selector for images', () =>
    matchesMarkup(getDisplayElementById(imageExamples, 'default'))
  )

  it('renders a file selector for images with error', () =>
    matchesMarkup(getDisplayElementById(imageExamples.state, 'file-selector-images-error'))
  )

  it('renders a file selector for images with dragover', () =>
    matchesMarkup(getDisplayElementById(imageExamples.state, 'file-selector-images-dragover'))
  )

  it('renders a file selector for images with dragover error', () =>
    matchesMarkup(getDisplayElementById(imageExamples.state, 'file-selector-images-dragover-error'))
  )
})

describe('Integrated File Selector', () => {
  it('renders a default integrated file selector', () =>
    matchesMarkup(getDisplayElementById(integratedExamples, 'default'))
  )

  it('renders an integrated file selector with drag', () =>
    matchesMarkup(getDisplayElementById(integratedExamples.state, 'integrated-file-selector-drag'))
  )

  it('renders an integrated file selector with dragover', () =>
    matchesMarkup(getDisplayElementById(integratedExamples.state, 'integrated-file-selector-dragover'))
  )

  it('renders an integrated file selector with dragover error', () =>
    matchesMarkup(getDisplayElementById(integratedExamples.state, 'integrated-file-selector-dragover-error'))
  )
})
