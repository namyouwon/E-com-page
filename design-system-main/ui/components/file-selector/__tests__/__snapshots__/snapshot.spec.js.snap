// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File Selector - Images renders a default file selector for images 1`] = `
<div
  className="slds-form-element"
>
  <span
    className="slds-form-element__label"
    id="file-selector-primary-label-17"
  >
    Attachment
  </span>
  <div
    className="slds-form-element__control"
  >
    <div
      className="slds-file-selector slds-file-selector_images"
    >
      <div
        className="slds-file-selector__dropzone"
      >
        <input
          accept="image/png"
          aria-describedby={null}
          aria-labelledby="file-selector-primary-label-17 file-selector-secondary-label18"
          className="slds-file-selector__input slds-assistive-text"
          id="file-upload-input-19"
          type="file"
        />
        <label
          className="slds-file-selector__body"
          htmlFor="file-upload-input-19"
          id="file-selector-secondary-label18"
        >
          <span
            className="slds-file-selector__button slds-button slds-button_neutral"
          >
            <svg
              aria-hidden={true}
              className="slds-button__icon slds-button__icon_left"
            >
              <use
                xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#upload"
              />
            </svg>
            Upload 
            Image
          </span>
          <span
            className="slds-file-selector__text slds-medium-show"
          >
            or Drop 
            Image
          </span>
        </label>
      </div>
    </div>
  </div>
</div>
`;

exports[`File Selector - Images renders a file selector for images with dragover 1`] = `null`;

exports[`File Selector - Images renders a file selector for images with dragover error 1`] = `null`;

exports[`File Selector - Images renders a file selector for images with error 1`] = `null`;

exports[`File Selector renders a default file selector 1`] = `
<div
  className="slds-form-element"
>
  <span
    className="slds-form-element__label"
    id="file-selector-primary-label-1"
  >
    Attachment
  </span>
  <div
    className="slds-form-element__control"
  >
    <div
      className="slds-file-selector slds-file-selector_files"
    >
      <div
        className="slds-file-selector__dropzone"
      >
        <input
          accept="image/png"
          aria-describedby={null}
          aria-labelledby="file-selector-primary-label-1 file-selector-secondary-label2"
          className="slds-file-selector__input slds-assistive-text"
          id="file-upload-input-3"
          type="file"
        />
        <label
          className="slds-file-selector__body"
          htmlFor="file-upload-input-3"
          id="file-selector-secondary-label2"
        >
          <span
            className="slds-file-selector__button slds-button slds-button_neutral"
          >
            <svg
              aria-hidden={true}
              className="slds-button__icon slds-button__icon_left"
            >
              <use
                xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#upload"
              />
            </svg>
            Upload 
            Files
          </span>
          <span
            className="slds-file-selector__text slds-medium-show"
          >
            or Drop 
            Files
          </span>
        </label>
      </div>
    </div>
  </div>
</div>
`;

exports[`File Selector renders a file selector dragover 1`] = `
<div
  className="slds-form-element"
>
  <span
    className="slds-form-element__label"
    id="file-selector-primary-label-9"
  >
    Attachment
  </span>
  <div
    className="slds-form-element__control"
  >
    <div
      className="slds-file-selector slds-file-selector_files"
    >
      <div
        className="slds-file-selector__dropzone"
      >
        <input
          accept="image/png"
          aria-describedby={null}
          aria-labelledby="file-selector-primary-label-9 file-selector-secondary-label10"
          className="slds-file-selector__input slds-assistive-text"
          id="file-upload-input-11"
          type="file"
        />
        <label
          className="slds-file-selector__body"
          htmlFor="file-upload-input-11"
          id="file-selector-secondary-label10"
        >
          <span
            className="slds-file-selector__button slds-button slds-button_neutral"
          >
            <svg
              aria-hidden={true}
              className="slds-button__icon slds-button__icon_left"
            >
              <use
                xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#upload"
              />
            </svg>
            Upload 
            Files
          </span>
          <span
            className="slds-file-selector__text slds-medium-show"
          >
            or Drop 
            Files
          </span>
        </label>
      </div>
    </div>
  </div>
</div>
`;

exports[`File Selector renders a file selector dragover with error 1`] = `
<div
  className="slds-form-element"
>
  <span
    className="slds-form-element__label"
    id="file-selector-primary-label-13"
  >
    Attachment
  </span>
  <div
    className="slds-form-element__control"
  >
    <div
      className="slds-file-selector slds-file-selector_files"
    >
      <div
        className="slds-file-selector__dropzone"
      >
        <input
          accept="image/png"
          aria-describedby={null}
          aria-labelledby="file-selector-primary-label-13 file-selector-secondary-label14"
          className="slds-file-selector__input slds-assistive-text"
          id="file-upload-input-15"
          type="file"
        />
        <label
          className="slds-file-selector__body"
          htmlFor="file-upload-input-15"
          id="file-selector-secondary-label14"
        >
          <span
            className="slds-file-selector__button slds-button slds-button_neutral"
          >
            <svg
              aria-hidden={true}
              className="slds-button__icon slds-button__icon_left"
            >
              <use
                xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#upload"
              />
            </svg>
            Upload 
            Files
          </span>
          <span
            className="slds-file-selector__text slds-medium-show"
          >
            or Drop 
            Files
          </span>
        </label>
      </div>
    </div>
  </div>
</div>
`;

exports[`File Selector renders a file selector with error 1`] = `
<div
  className="slds-form-element"
>
  <span
    className="slds-form-element__label"
    id="file-selector-primary-label-5"
  >
    Attachment
  </span>
  <div
    className="slds-form-element__control"
  >
    <div
      className="slds-file-selector slds-file-selector_files"
    >
      <div
        className="slds-file-selector__dropzone"
      >
        <input
          accept="image/png"
          aria-describedby={null}
          aria-labelledby="file-selector-primary-label-5 file-selector-secondary-label6"
          className="slds-file-selector__input slds-assistive-text"
          id="file-upload-input-7"
          type="file"
        />
        <label
          className="slds-file-selector__body"
          htmlFor="file-upload-input-7"
          id="file-selector-secondary-label6"
        >
          <span
            className="slds-file-selector__button slds-button slds-button_neutral"
          >
            <svg
              aria-hidden={true}
              className="slds-button__icon slds-button__icon_left"
            >
              <use
                xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#upload"
              />
            </svg>
            Upload 
            Files
          </span>
          <span
            className="slds-file-selector__text slds-medium-show"
          >
            or Drop 
            Files
          </span>
        </label>
      </div>
    </div>
  </div>
</div>
`;

exports[`Integrated File Selector renders a default integrated file selector 1`] = `
<div
  className="slds-file-selector slds-file-selector_integrated slds-file-selector_integrated"
>
  <div
    aria-hidden="true"
    className="slds-file-selector__dropzone slds-file-selector__dropzone_integrated"
  >
    <input
      accept="image/png"
      className="slds-file-selector__input slds-assistive-text"
      id="file-upload-input-01"
      tabIndex="-1"
      type="file"
    />
    <label
      className="slds-file-selector__body slds-file-selector__body_integrated"
      htmlFor="file-upload-input-01"
    >
      <svg
        aria-hidden={true}
        className="slds-file-selector__body-icon slds-icon slds-icon-text-default"
      >
        <use
          xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#upload"
        />
      </svg>
      <span
        className="slds-file-selector__text slds-file-selector__text_integrated slds-text-heading_medium slds-text-align_center"
      >
        Drop Files
      </span>
    </label>
  </div>
  Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
</div>
`;

exports[`Integrated File Selector renders an integrated file selector with drag 1`] = `null`;

exports[`Integrated File Selector renders an integrated file selector with dragover 1`] = `null`;

exports[`Integrated File Selector renders an integrated file selector with dragover error 1`] = `null`;
